octobot_trading/exchanges/exchanges.py

```diff
@@ -22,7 +22,7 @@ class ExchangeConfiguration:
     def __init__(self, exchange_manager):
         self.exchange_manager = exchange_manager
         self.exchange_name = exchange_manager.exchange.name
-        self.id = exchange_manager.id
```

Comment:
Why removing self.id ?

Reply:
Hum I don't understand... I missed something with git probably

Reply:
Thanks, it's fixed now.